test/sequential/test-async-wrap-getasyncid.js

```diff
@@ -269,7 +269,7 @@ if (common.hasCrypto) { // eslint-disable-line node-core/crypto-check
       }
       testInitialized(wreq, 'WriteWrap');
     }
-    req.address = common.localhostIPv4;
+    req.address = common.localhostIP;
```

Comment:
If we want to use IPv6 here, we need to use `handle.connect6` instead of `handle.connect` on line 274, and specify `common.localhostIP` instead of `common.localhostIPv4` on line 241.